gpflow/likelihoods/likelihoods.py

```diff
@@ -310,60 +524,61 @@ def __init__(self, invlink=tf.exp, **kwargs):
         self.invlink = invlink
         self.shape = Parameter(1.0, transform=positive())
 
-    def log_prob(self, F, Y):
+    def _scalar_density(self, F, Y):
         return logdensities.gamma(Y, self.shape, self.invlink(F))
 
-    def conditional_mean(self, F):
+    def _conditional_mean(self, F):
         return self.shape * self.invlink(F)
 
-    def conditional_variance(self, F):
+    def _conditional_variance(self, F):
         scale = self.invlink(F)
         return self.shape * (scale ** 2)
 
-    def variational_expectations(self, Fmu, Fvar, Y):
+    def _variational_expectations(self, Fmu, Fvar, Y):
         if self.invlink is tf.exp:
-            return (
+            return tf.reduce_sum(
                 -self.shape * Fmu
                 - tf.math.lgamma(self.shape)
                 + (self.shape - 1.0) * tf.math.log(Y)
-                - Y * tf.exp(-Fmu + Fvar / 2.0)
+                - Y * tf.exp(-Fmu + Fvar / 2.0),
+                axis=-1,
             )
         else:
-            return super().variational_expectations(Fmu, Fvar, Y)
+            return super()._variational_expectations(Fmu, Fvar, Y)
 
 
-class Beta(Likelihood):
+class Beta(ScalarLikelihood):
     """
     This uses a reparameterisation of the Beta density. We have the mean of the
     Beta distribution given by the transformed process:
 
-        m = sigma(f)
+        m = σ(f)
```

Comment:
is this common notation for inv_probit ? would it be clearer to just say "invlink" here (which is what the argument to the constructor is called)?